src/actions/tags.js

```diff
@@ -112,10 +112,16 @@ export const clearSearchString = () => {
 * Action to reset all tags to empty
 */
 
-export const resetTags = (displayName, id) => {
```

Comment:
So it is. I'd be tempted to fix that and try to get down to one single place where that's defined though, rather than letting it continue to spread.